app/assets/stylesheets/type.css

```diff
@@ -46,7 +46,7 @@ label.t-hidden {
   text-transform: uppercase;
   color: #e9573f; }
   .t-list__heading:not(:first-child) {
-    margin-top: 25px; }
+    margin-top: 30px; }
```

Comment:
1. usually 30px or 32px is used as a "gutter"
2. for some mysterious reasons 25px makes render problems (you can see some of the at "before" screenshot as well), this fixes misalignment finally